agent/bench-scripts/test-bin/uperf

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "$0 $*" >> $_testlog
+
+# Mock uperf command behaviors.
+if [[ "${1}" == "-V" ]]; then 
+	echo "Uperf Version ${_BM_UPERF_VER:-1.0.42}"
+	echo "... garbage to be ignored ..."
+fi
```

Comment:
Based on what you said about the analogous portion of `agent/bench-scripts/test-bin/fio`, we should be invoking `exit 0` at the end of the `if` statement, here, instead of falling through to the common exit point which might not exit "successfully".